docs/azmcp-commands.md

```diff
@@ -193,10 +193,19 @@ azmcp keyvault key create --subscription <subscription> --vault <vault-name> --k
 ### Azure Service Bus Operations
 ```bash
 # Peeks at messages in a Service Bus queue
-azmcp servicebus queue peek --subscription <subscription> --namespace <service-bus-namespace> --queue <queue-name> [--max-messages <int>]
+azmcp servicebus queue peek --subscription <subscription> --namespace <service-bus-namespace> --queue-name <queue-name> [--max-messages <int>]
 
-# Peeks at messages in a Service Bus topic/subscription
-azmcp servicebus topic peek --subscription <subscription> --namespace <service-bus-namespace> --topic-name <topic-name> --subscription-name <subscription-name> [--max-messages <int>]
+# Returns runtime and details about the Service Bus queue
+azmcp servicebus queue details --subscription <subscription> --namespace <service-bus-namespace> --queue-name <queue-name>
+
+# Gets runtime details a Service Bus topic
+azmcp servicebus topic details --subscription <subscription> --namespace <service-bus-namespace> --topic-name <topic-name>
+
+# Peeks at messages in a Service Bus subscription within a topic.
+azmcp servicebus topic subscription peek --subscription <subscription> --namespace <service-bus-namespace> --topic-name <topic-name> --subscription-name <subscription-name> [--max-messages <int>]
```

Comment:
Please run manual test to see if the model can distinguish between subscription and subscription-name, wondering if we need to have a more verbose name for subscription name of if the tool desc is enough.

Reply:
Good catch.

Reply:
I tried and it's smart enough to distinguish. :)